src/intrinsics/fb-www/global.js

```diff
@@ -97,18 +107,58 @@ export default function(realm: Realm): void {
           return reactRelay;
         }
         return realm.fbLibraries.reactRelay;
+      } else {
+        let requireVal;
+
+        if (realm.fbLibraries.other.has(requireNameValValue)) {
+          requireVal = realm.fbLibraries.other.get(requireNameValValue);
+        } else {
+          requireVal = createAbstract(realm, "function", `require("${requireNameValValue}")`);
+          realm.fbLibraries.other.set(requireNameValValue, requireVal);
+        }
+        invariant(requireVal instanceof AbstractValue);
+        return requireVal;
+      }
+    }),
+    writable: true,
+    enumerable: false,
+    configurable: true,
+  });
+
+  global.$DefineOwnProperty("cx", {
+    value: new NativeFunctionValue(realm, "cx", "cx", 0, (context, [cxString]) => {
+      invariant(cxString instanceof StringValue);
+      let cxValue = `Bootloader("${cxString.value}")`;
+      let cx;
+
+      if (realm.fbLibraries.other.has(cxValue)) {
+        cx = realm.fbLibraries.other.get(cxValue);
+      } else {
+        cx = createAbstract(realm, "function", cxValue);
+        realm.fbLibraries.other.set(cxValue, cx);
+      }
+      invariant(cx instanceof AbstractValue);
+      return cx;
+    }),
+    writable: true,
+    enumerable: false,
+    configurable: true,
+  });
+
+  global.$DefineOwnProperty("Bootloader", {
+    value: new NativeFunctionValue(realm, "Bootloader", "Bootloader", 0, (context, [bootloaderString]) => {
+      invariant(bootloaderString instanceof StringValue);
+      let bootloaderValue = `Bootloader("${bootloaderString.value}")`;
```

Comment:
Bootloader isn't called as a function, it's called like `Booloader.loadAllModules()` or something (see the bundles)